docs/guides/run_a_job.md

```diff
@@ -0,0 +1,227 @@
+packages you want available on the server that runs your code. This is not a
+specification between a CPU server and a GPU server (that's the
+[Instance Type](#instance-type) `OPTION` of `floyd run`).
+
+FloydHub offers servers with many different deep-learning software packages
+pre-installed.  You can find a list of all the available environments
+[here](../guides/environments).
+
+Use the `--env` flag to specify which environment you would like your job to
+run in. It is best practice to pass the entire name of the environment,
+including the version number, to the `--env` flag. For example, instead of
+`--env tensorflow`, use `--env tensorflow-1.3`.
+
+### Message
+
+Using `--message` or `-m`, you can specify a message that describes your job,
+similiar to the way a commit message describes a git commit. The job message
+will be displayed at various places on floydhub.com and is useful when
+reviewing past jobs that you'd like to iterate on.
+
+Example:
+
+```
+$ floyd run -m "lorem ipsum" "python my_first_script.py"
+Creating project run. Total upload size: 195.0B
+Syncing code ...
+[================================] 1254/1254 - 00:00:00
+
+JOB NAME
+--------------------------------
+mckay/projects/message-project/1
+
+To view logs enter:
+   floyd logs mckay/projects/message-project/1
+```
+
+Here are some examples of where the job message will be displayed on floydhub.com:
+
+![Job Message on Job Detail Page](../img/job_detail_message.jpg)
+![Job Message on Project Detail Page](../img/project_detail_job_message.jpg)
+
+### Tensorboard
+
+The `--tensorboard` flag allows you to enable Tensorboard on your job. For more details, see [this article](../guides/jobs/tensorboard).
+
+## [COMMAND]
+
+The `[COMMAND]` portion of `floyd run` is the command that will be executed on
+the server when your job begins. It will be run in the directory on the server
+that holds your code. To decide what to put in the `[COMMAND]`, answer this
+question:
+
+> What command would I execute to kick off my code locally?
+
+The answer to that question is what you should put into the `[COMMAND]` portion
+of `floyd run`.
+
+Any valid `bash` command will work in the `[COMMAND]` portion of `floyd run`.
+For example, try these simple examples and look at their logs:
+
+```
+$ floyd run "pwd"
+```
+
+```
+$ floyd run "ls"
+```
+
+```
+$ floyd run "python -v"
+```
+
+```
+$ floyd run "echo 'Hello, world!'"
+```
+
+Most commonly, you'll be kicking off a Pyton script in your `[COMMAND]` with something like:
+
+```
+$ floyd run "python train.py"
+```
+
+But you can feel free to get creative!
+
+!!! important "Pro Tip"
+
+    Try chaining together multiple commands with `&&` like this:
+
+    ```
+    $ floyd run "bash my_setup_script.sh && python train.py"
+    ```
+
+    For more examples, check out our
+    [symlinking tutorial](../guides/data/symlink_mounted_data/#1-using-the-command-portion-of-floyd-run).
+
```

Comment:
A box with a table with a few examples of the common commands at the end would be useful:

- floyd run "python train.py"
- floyd run --env tensorflow-0.12 "python train.py"
- floyd run --env pytorch-0.2 --mode jupyter
- floyd run --env pytorch-0.2 --mode jupyter --data a/datasets/b/1:mount
- floyd run --mode serve
- etc.